share/revng/pipelines/revng-pipelines.yml

```diff
@@ -47,6 +47,12 @@ Containers:
     Type: clift-module
   - Name: model-type-definitions.tar.gz
     Type: model-type-definitions
+  - Name: new-types-and-globals.h
+    Type: new-model-header
+  - Name: new-helpers.h
+    Type: new-helper-header
+  - Name: new-type-definitions.tar.gz
+    Type: new-type-definitions
```

Comment:
Same naming problem here about `new`. Please adopt the `legacy` conventions on the old stuff instead.

Reply:
I'm gonna make other individual small comments wherever I see this problem, but don't count on my comments to be exhaustive. Double check yourself too.